server/FeatureExtraction/FeatureExtraction.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<executable>
+  <category>HistomicsTK</category>
+  <title>Performs Feature Extraction</title>
+  <description>Extracts initial features from an image</description>
+  <version>0.1.0</version>
+  <documentation-url>https://histomicstk.readthedocs.org/en/latest/</documentation-url>
+  <license>Apache 2.0</license>
+  <contributor>Sanghoon Lee (Emory University)</contributor>
+  <acknowledgements>This work is part of the HistomicsTK project.</acknowledgements>
+  <parameters>
+    <label>IO</label>
+    <description>Input/output parameters.</description>
+    <image>
+      <name>inputImageFile</name>
+      <label>Input Image</label>
+      <channel>input</channel>
+      <index>0</index>
+      <description>Input image</description>
+    </image>
+    <double-vector>
+      <name>stainColor_1</name>
+      <label>Color of stain-1</label>
+      <channel>input</channel>
+      <index>1</index>
+      <description>RGB Color of stain-1</description>
+    </double-vector>
+    <double-vector>
+      <name>stainColor_2</name>
+      <label>Color of stain-2</label>
+      <channel>input</channel>
+      <index>2</index>
+      <description>RGB Color of stain-2</description>
+    </double-vector>
+    <file>
+      <name>outputFile</name>
+      <label>Output HDF5 file</label>
+      <channel>output</channel>
+      <index>3</index>
+      <description>Output HDF5 file</description>
+    </file>
+    <double-vector>
+      <name>stainColor_3</name>
+      <label>Color of stain-3</label>
+      <description>RGB Color of stain-3</description>
+      <longflag>stainColor_3</longflag>
+      <default>0,0,0</default>
+    </double-vector>
+    <integer>
+      <name>K</name>
+      <label>K</label>
+      <description>Number of points for boundary resampling</description>
+      <longflag>k</longflag>
+      <default>128</default>
+    </integer>
+    <integer>
+      <name>Fs</name>
+      <label>Fs</label>
+      <description>Number of frequency bins for calculating FSDs</description>
+      <longflag>fs</longflag>
+      <default>6</default>
+    </integer>
+    <integer>
+      <name>Delta</name>
+      <label>Delta</label>
+      <description>Used to dilate nuclei and define cytoplasm region</description>
+      <longflag>delta</longflag>
+      <default>8</default>
+    </integer>
+  </parameters>
```

Comment:
@slee172 Try grouping the parameters. 

For example you can:

* Put `stainColor_1`, `stainColor_2`, and `stainColor_3` into a group named `Color Deconvolution Parameters`. 
* Put `K`, `Fs`, and `Delta` into a group called `Feature Extraction Parameters`
* Put `inputImageFile` and `outputFile` into a group called `IO`
* Also expose nuclei segmentation parameters and create a separate group for them.

When there are a large number of parameters, grouping helps keep it modular and less overwhelming.

See [this](https://github.com/DigitalSlideArchive/HistomicsTK/blob/nuclei_segmentation_cli/server/NucleiSegmentation/NucleiSegmentation.xml#L74-L112) for an example.

Also, if you think a certain group of parameters is meant to be tweaked only by expert users, then you make that group by setting the `advanced` attribute of parameters to `true` as follows:
```
<parameters advanced="true">
</parameters>
```
This will make group panel closed/hidden in the GUI by default.